.github/workflows/ci.yml

```diff
@@ -242,6 +242,8 @@ jobs:
       - name: Execute registry-certs test (Fork Only)
         run: ./tests/registry-certs/test.sh --build-arg DOCKERHUB_AUTH=false
         if: github.event_name != 'push' && github.event.pull_request.head.repo.full_name != github.repository
+      - name: Execute remote buildkit test
```

Comment:
Minor comment here on wording -- since we're still using localhost I the "remote" is a bit misleading. Maybe this test should be named something like `Execute local buildkit with mTLS test` or something like that